packages/theming/atlas/scripts/build.js

```diff
@@ -63,12 +64,18 @@ async function buildAndCopyAtlas(watchMode, destination) {
                 {
                     name: "native-design-properties-and-manifest",
                     command: `copy-and-watch ${watchArg} "src/themesource/atlas_core/native/**/*.json" "${destination}/themesource/atlas_core/native"`
+                },
+                {
+                    name: "atlas-2-to-3-migration-tool",
+                    command: `copy-and-watch ${watchArg} "src/themesource/atlas_core/migration/*" "${destination}/themesource/atlas_core/migration"`
                 }
             ],
             {
                 killOthers: ["failure"]
             }
         );
+        // This needs to be done after the concurrently batch above, since we're dependent on the resulting `theme` folder.
+        await zip(`${destination}/theme`, `${destination}/themesource/atlas_core/migration/theme.zip`);
```

Comment:
Should I add any logs to this?

Reply:
At least output an error when zipping fails.

Reply:
The outer `try/catch` should catch an error and log it appropriately I think

Reply:
What goes into `theme.zip`, it looks like it will just be `web` and `native`, but we want `theme/web|native`, right?

Reply:
@Keraito True. Let's not add any extra log messages.

Reply:
Lemi double check that @jordanch

Reply:
ok so I checked and @jordanch is correct. I tried getting the `theme` folder automatically into the zip, but it would also include `themesource`, which means that our script would also totally override the user's `themesource` when the tool is run. I assume this is a no-no?

Reply:
No-no, indeed.

What about in `.cmd` make `theme`, then here copy over `web` + `native` into `theme`?

Reply:
I like your suggestion.

Reply:
f67657c